### DISCUSSION TITLE
compfy (comma point)

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#435] [Q&A]
Is it you or just compfy that only one point is accepted?

**Comment by mcmonkey4eva** [comment]
what

**Comment by kalle07** [comment]
![grafik](https://github.com/user-attachments/assets/42c98542-5ee9-43be-bf38-452a226d7e98)

so here only point is accepted  (.)
should i ask on compfy repro if they can accept comma  (,)

**Reply by mcmonkey4eva** [reply]
That is ComfyUI yeah.

And: `.` for decimal is international standard, the `,` decimal is used historically in a few countries but generally is considered a historical localization thing, not relevant to the modern technological world.

**Comment by kalle07** [comment]
I will tell you that Germany is one of them  ;)
